### ISSUE TITLE
ColumnPermutation getter and setter in TableImpl (org.jowidgets.impl.widgets.basic) are inconsistent

### ISSUE THREAD

**Issue by lack3r** [jo-source/jo-widgets#72]
Hi.

Calling `getColumnPermutation` after calling `setColumnPermutation` gives us a list with a different order of column permutations than the ones we set.

> Steps to reproduce:

Let's suppose the table has three columns:

And we want to set the following column permutation:

```java
List<Integer> columnPermutations = new ArrayList<>();
columnPermutations.add(2);
columnPermutations.add(0);
columnPermutations.add(1);
```

We call `setPermutation` with our list that contains the numbers [2,0,1] as below.
`TableImpl.setPermutation(columnPermutations);`
If we call 
`TableImpl.getPermutation()`
We get a list containing the values [0,1,2] instead of the ones we set before.

Expected behavior:
`setPermutation` sets the permutation correctly in the TableImpl and `getPermutation`gets the permutations correctly from the TableImpl.
In this case if we set [2,0,1] as column permutations, we expect to get a list with the same values when calling the `getPermutation`

The following code is from `TableImpl`(org.jowidgets.impl.widgets.basic)

```java
    @Override
    public ArrayList<Integer> getColumnPermutation() {
        return modelSpiAdapter.getCurrentPermutation();
    }

    @Override
    public void setColumnPermutation(final List<Integer> permutation) {
        getWidget().setColumnPermutation(modelSpiAdapter.convertColumnPermutationToView(permutation));
    }
```

So we can see that:

1. The `setColumnPermutation` sets the permutation in the `getWidget()` and the state of the `modelSpiAdapter` does not change, while 
2. The `getColumnPermutation` returns the permutation from the `modelSpiAdapter`.

**Comment by herrgrossmann** [comment]
Fix will be available with version 0.73.0

**Comment by herrgrossmann** [comment]
This problem has only been occurred with nat table spi impl
